go-controller/pkg/ovn/ovn.go

```diff
@@ -119,9 +142,153 @@ func (oc *Controller) Run() error {
 			return err
 		}
 	}
+
+	if config.EmptyLbEvents || config.Kubernetes.OVNEmptyLbEvents {
+		go oc.eventChecker(stopChan)
```

Comment:
@putnopvut can we rename this to `ovnControllerEventChecker`? I know that might be a bit redundant with the type of `oc` but they refer to different things.